harmony-fe/src/components/postContainer/postContainer.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import Post from '../post/post'
+import Link from 'next/link'
+
+interface PostContainerProps {
+  contentType: string
+}
+export default function PostContainer({ contentType }: PostContainerProps) {
+  return (
+    <div>
+      <h2>
+        These are your {contentType} Posts. For now they are static but later
+        they will be dynamic
+      </h2>
+      {/* <Link href="./posts/[postId].tsx"> */}
```

Comment:
Also could we remove the comments here and a few lines below this one?

Reply:
Updated now.